### DISCUSSION TITLE
Swift Support

### DISCUSSION THREAD

**Discussion by jorgeorejas** [supabase/supabase#1871] [Questions] [answered]
I've found some references to the Supabase + Swift integration but there is no mention to it in the web although I found a link. Does someone knows when it will be public? I'm currently working on my web SaaS but the next step would be native iOS/iPadOS and MacOS support.

**Comment by inian** [comment, marked as answer]
Hey @jorgeorejas, we have launched the Swift libraries as part of our May monthly update. More info [here](https://supabase.io/blog/2021/06/02/supabase-beta-may-2021#new-swift-libraries)

**Reply by jorgeorejas** [reply]
Thank you!! Will a more detailed documentation arrive soon?
